src/dash-adapter.ts

```diff
@@ -1312,13 +1320,37 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
         error = error.data[0];
         this._requestFilterError ? (this._requestFilterError = false) : (this._responseFilterError = false);
       }
+      if (error.code === 1002) {
+        error.severity = this._onError1002ChangeSeverity() ? Error.Severity.CRITICAL : error.severity;
+      }
       this._trigger(EventType.ERROR, new Error(error.severity, error.category, error.code, error.data));
       DashAdapter._logger.error(error);
+
       if (error.severity === Error.Severity.CRITICAL) {
         this.destroy();
       }
     }
   }
+  private _onError1002ChangeSeverity() {
```

Comment:
the method determines if severity should change, right ? so it should probably be called something like "shouldError1002ChangeSeverity"